site/mkdocs.yml

```diff
@@ -23,33 +23,37 @@ theme:
   logo: assets/images/iceberg-logo-icon.png
   favicon: assets/images/favicon-96x96.png
   features:
+    - content.tabs.link
+    - content.code.copy
+    - content.code.annotate
     - navigation.tabs
     - navigation.path
     - navigation.top
     - navigation.tracking
-    - toc.follow
+    - offline
     - search.suggest
     - search.highlight
-    - content.tabs.link
-    - content.code.copy
-    - content.code.annotate
+    - toc.follow
 
 plugins:
   - search
   - macros:
       include_yaml:
         - variables.yml
   - monorepo
+  - offline:
+      enabled: !ENV [OFFLINE, false]
 
 nav:
   - Quickstart:
     - Spark: spark-quickstart.md
     - Hive: hive-quickstart.md
   - Docs: 
     - nightly: '!include docs/docs/nightly/mkdocs.yml'
-#    - latest: '!include docs/docs/latest/mkdocs.yml'
-#    - 1.3.1: '!include docs/docs/1.3.1/mkdocs.yml'
-#    - 1.3.0: '!include docs/docs/1.3.0/mkdocs.yml'
+    - latest: '!include docs/docs/latest/mkdocs.yml'
```

Comment:
I was looking at this:

<img width="1524" alt="image" src="https://github.com/apache/iceberg/assets/1134248/c15926b7-9d7c-4017-9ece-819bea66c930">

It looks a bit odd since 1.4.2 is the same as latest. Do we really need this, it is quite obvious that 1.4.2 is the latest.

Reply:
I've gotten mixed signals depending who I've spoken to about meta refresh for redirects.

According to most of the stuff that comes up it's [not as big of a deal](https://webmasters.stackexchange.com/a/141240) as some made it out to be. So we can likely change `latest` to being a refresh page, but I'm curious to see how mkdocs renders that.

We should ideally have latest get ranked by Google and we may just want to be cognizant if we get a bunch of random versions or if latest gets ranked higher.

It's a bit of lore and black box when it comes to ensuring a certain version gets indexed and ranked by Google. No real way to control it so this was my feeble attempt.

<https://developers.google.com/search/docs/crawling-indexing/301-redirects#metarefresh>

Based on their docs, I think we should be fine if we update all the pages to set the [`site_url`](https://www.mkdocs.org/user-guide/configuration/#site_url) which will always point to latest. 

Also, we'll likely want to add an [outdated version warning](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/#version-warning) which will link to the "latest" equivalent of the page if available.

Reply:
For me I was wondering why "nightly" was first isn't that our "unstable" basically? 

I agree ideally we have something like 1.4.2 (latest)

Reply:
Yeah, so nightly is unstable. I thought it would be best to have it up top for the easiest access. Perhaps this ordering?:

```
1.4.2 (latest)
nightly
1.4.1
1.4.0
...
```